### ISSUE TITLE
Cleanup: stats

### ISSUE THREAD

**Issue by cafehaine** [prologin/metalang#22]
The statics are cool, but aren't used anywhere (btw, [here](http://eelte.megami.fr/) the repo link is outdated).

Also they use the /out folder, which really should'nt be commited

I see two options: 

- move the scripts to a subdirectory, and rewrite them to no longer depend on the /out folder
- remove the scripts entirely from the repo, since they aren't used

**Comment by coucou747** [comment]
out is the only way to keep a track of printers.
when we improve printers, it's powerfull to see the diff in out directory.

I understand that it's dirty to have generated files into git, but I didn't saw another solution.

**Comment by Zopieux** [comment]
I agree it's important to have golden references of how things should look like. When you update the printers, golden have diffs. Check that the diffs are what you intended, then just update the goldens and commit that as part of the change.

**Comment by coucou747** [comment]
it's the way out/ works :)

**Comment by Zopieux** [comment]
I know, I was expali
